Java/NewAppium_SampleProjects/Chapter9-SampleChatrApplication/src/com/pCloudy/testNG/TestCase.java

```diff
@@ -3,31 +3,39 @@
 import java.io.IOException;
 
 import org.openqa.selenium.By;
-import org.openqa.selenium.OutputType;
-import org.openqa.selenium.TakesScreenshot;
+
+import org.openqa.selenium.WebElement;
 import org.testng.annotations.AfterMethod;
 import org.testng.annotations.Test;
 
+import io.appium.java_client.AppiumDriver;
+
 public class TestCase extends Runner{
 	
+	public  AppiumDriver<WebElement> Hostdriver;
+	public  AppiumDriver<WebElement> Childdriver;
+	
 	@Test
 	public void Test() throws IOException, InterruptedException {     
 
-		System.out.println(driverAppium.size());
-		driverAppium.get(0).findElement(By.xpath("//android.widget.Button[@resource-id='com.pcloudy.appiumdemo:id/accept']")).click(); 
+		Hostdriver=driverlistAppium.get(0);
```

Comment:
@SridattaRishi still the list has been used. Please access the driver through some unique value like deviceName and all